frontend/src/reviews/ReviewItem.tsx

```diff
@@ -0,0 +1,46 @@
+import { ListItem, ListItemButton, Stack, Typography } from '@mui/material'
+import { useRouter } from 'next/router'
+import { ReviewRequestInterface } from 'types/interfaces'
+import { timeDifference } from 'utils/dateUtils'
+
+type ReviewItemProps = {
+  review: ReviewRequestInterface
+}
+
+export default function ReviewItem({ review }: ReviewItemProps) {
+  const router = useRouter()
+
+  function handleListItemClick() {
```

Comment:
This function has been changed, the rest of the file is the same as it was when the component was in `ReviewsList.tsx`